.github/workflows/pr_push.yml

```diff
@@ -141,7 +141,10 @@ jobs:
       run: cmake --build build --target clang-format-check
 
     - name: Check CMake formatting
-      run: cmake --build build --target cmake-format-check
+      run: |
+        cmake --build build --target cmake-format-apply
+        echo "Errors found:"
```

Comment:
heh, is this `echo` really needed? now we print this, even though there were no errors

if you want this message printed it should be under `if` or e.g. like this:
`git diff --exit-code && echo "Errors found! || echo "positive message"`

Reply:
I agree that this message is not needed at all.

Reply:
Fail + diff output is sufficient. Echo removed.